components/cache/cache_invalidation.rst

```diff
@@ -7,7 +7,7 @@ Cache Invalidation
 
 Cache invalidation is the process of removing all cached items related to a
 change in the state of your model. The most basic kind of invalidation is direct
-items deletion. But when the state of a primary resource has spread across
+item deletion. But when the state of a primary resource has spread across
```

Comment:
Not sure this change is correct 

cc @nicolas-grekas @Nyholm

Reply:
It can either be stated as:

- ... is direct item deletion, or
- ... is the direct deletion of items.